### DISCUSSION TITLE
Where's the tag picker ?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#275] [Q&A] [answered]
i think there used to be a tag picker or plans for tag picker (creatable i.e) but then never got to see it.  Are there any plans for multi select/select to have a "creatable" variant ? 

![tag-picker](https://user-images.githubusercontent.com/5988991/135754339-cabed292-27d3-4444-ac15-7eb8b2467b45.gif)

**Comment by rtivital** [comment, marked as answer]
Yes creatable options is coming to Select and MutiSelect components – https://github.com/mantinedev/mantine/pull/269
TagPicker was hard to maintain and customize, that's why it was deprecated. Maybe it will be added back as part of @mantine/core package.

**Reply by dukesx** [reply]
Well i am anxious to see this package back again. Looks state of the art. At least i haven't seen anything more advanced than this.

**Comment by trompx** [comment]
The UI of the old tag picker was better than the creatable implemented in the select/multiselect. Would also like to see this package back again.

**Comment by paulm17** [comment]
Wow that tag picker is one of the best that I've seen so far.  😮

**Comment by chinnuu05** [comment]
@rtivital any chance we can get the old tag picker back as an extension or add on package?
